Week3/homework/3-step3.js

```diff
@@ -1,25 +1,61 @@
 'use strict';
 
 // use a 'for' loop
-function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
-  return str;
+'use strict'
+
+function repeatStringNumTimes(str, num) {
+    var repeat = "";
+
+    for (var i = 0; i < num; i++) {
+
+        var x = repeat += str;
```

Comment:
In my re-reviewing your homework, you don't need this `x` variable here, you can do't with the `repeat` variable it self